src/backend/cuda/CMakeLists.txt

```diff
@@ -301,13 +294,19 @@ if(UNIX)
       -Wl,--start-group
       ${CUDA_culibos_LIBRARY} #also a static libary
       ${CUDA_cublas_static_LIBRARY}
-      ${CUDA_cublasLt_static_LIBRARY}
       ${CUDA_cufft_static_LIBRARY}
-      ${CUDA_lapack_static_LIBRARY}
       ${CUDA_cusparse_static_LIBRARY}
       ${cusolver_static_lib}
       -Wl,--end-group
   )
+
+  if(CUDA_VERSION VERSION_GREATER 9.5)
+    target_link_libraries(af_cuda_static_cuda_library
+      PRIVATE
+        ${CUDA_cublasLt_static_LIBRARY}
+        ${CUDA_lapack_static_LIBRARY})
```

Comment:
Makes sense, these are not there prior to toolkit 10 -  I should have thought about this